manifests/webclient.pp

```diff
@@ -31,6 +33,18 @@
     content => template('i2b2/webclient_config_data.js.erb'),
   }
 
+  file { "$webclient_dir/cmi_data_portal_override.php" :
+    ensure => file,
+    source => 'puppet:///modules/i2b2/cmi_data_portal_override.php',
+  }
+
+  exec { 'insert-css' :
+    cwd     => $webclient_dir,
+    command => "sed -i '$css_sed_expr' default.htm",
+    unless  => "grep '$css_declaration' default.htm",
+    require => Exec[ "extract-$webclient_zip" ],
+  }
```

Comment:
Since we're building an i2b2 module, not a CMI i2b2 module, the location of the CSS file should be configurable and optional and `cmi_data_portal_override.php` should be renamed to sth more generic.